src/Component/Filter/ComparisonFilter/ComparisonFilterUi.css

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Please use less instead of css

Reply:
OK...

Reply:
I created a separate issue for this (#65), so we can tackle this later (since this seems a bit more complicated that one might think at first sight).